README.md

```diff
@@ -7,3 +7,145 @@ Derived from IDS Usage Control App by Fraunhofer IESE
 [![Docker badge](https://img.shields.io/docker/pulls/rdlabengpa/ids_uc_data_app.svg)](https://hub.docker.com/r/rdlabengpa/ids_uc_data_app)
 <br/>
 
+The TRUE Connector Usage Control Data App, based on IDS Usage Control App by Fraunhofer IESE enabling:
+
+- provides access to a resource with specific identifier defined by the target property or rule in a specific time interval
+
+## How to Configurate
+
+The configuration should be performed in the application.yml file:
+
```

Comment:
Maybe to provide information that default values are the following:....
Since UC data app should be ready for usage "out of the box"
If someone wants to tweak it - application.yml file is starting point, and then explain properties of interest